framework/workspace/qml/MuseScore/Workspace/WorkspacesControl.qml

```diff
@@ -0,0 +1,27 @@
+import QtQuick 2.7
```

Comment:
please bring to the same version in all new files

Reply:
fixed